src/core/utils.spec.ts

```diff
@@ -603,6 +604,21 @@ describe('potion/core', () => {
                 expect(findPotionResource('/bar', resources)).toBeUndefined();
                 expect(findPotionResource('/foo/1', resources)).toEqual(res);
             });
+            it('should return a Potion resource even when the resource name is a substring of another resource name', () => {
+                const resources = {
+                    '/foo': Foo,
+                    '/foo_bar': FooBar
+                };
+
+                expect(findPotionResource('/foo_bar/1', resources)).toEqual({
+                    resourceURI: '/foo_bar',
+                    resource: FooBar
+                });
+                expect(findPotionResource('/foo/1', resources)).toEqual({
```

Comment:
Could you add `findPotionResource('/foo', resources) -> Foo` and `findPotionResource('/foo_bar', resources) -> FooBar` as well?

Reply:
Yes - will add these now.